sources/regex.go

```diff
@@ -28,6 +28,48 @@ func init() {
 			"socks4": "/api/v1/get?type=socks4",
 			"socks5": "/api/v1/get?type=socks5",
 		}, true, 3*time.Hour),
+		regexSource(56, "https://freeproxychecker.com/result", ":", map[string]string{
+			"http":   "/socks4_proxies.txt",
```

Comment:
Move "/result" here, so that it's possible to get to homepage of the source from the dashboard